src/StudioCore/Assets/Paramdex/ER/Names/NpcParam.txt

```diff
@@ -179,7 +179,6 @@
 21910000 Radagon - Crucified (Unscaled)
 22000000 Elden Beast
 22000078 Elden Beast
-22000178 Elden Beast
```

Comment:
I'm guessing removals in this file are because they don't appear, than that they don't match chr-id conventions?